ui-kit/HeroCardGrid/HeroCardGrid.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { systemPropTypes } from '..';
+import Styled from './HeroCardGrid.styles';
+import { indexOf } from 'lodash';
+
+function HeroCardGrid(props = {}) {
+  return <Styled {...props}>
+    {props.children.map((n, i) => {
+    if(i < 1) 
+      return <Styled.LargeGridItem key={i}>{n}</Styled.LargeGridItem>
+    else 
+      return n
+    })}
+  </Styled>;
```

Comment:
What do you think about doing it like this?

```js
function HeroCardGrid(props = {}) {
  function render() {
    return props.children.map((n, i) => {
      if (i < 1) return <Styled.LargeGridItem key={i}>{n}</Styled.LargeGridItem>;
      else return n;
    });
  }

  return <Styled {...props}>{render()}</Styled>;
}
```

Reply:
Yeah I like that!